### PR TITLE
[zh] Translate kubelet config v1 reference

### PR DESCRIPTION
**Pull request by helen-frank**

### CODE REVIEW

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+   </p>
+</td>
+</tr>
+</tbody>
+</table>
+
+## `CredentialProvider`     {#kubelet-config-k8s-io-v1-CredentialProvider}
+
+
+**Appears in:**
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kubelet-config.v1.md:70]: 翻译

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+- [CredentialProviderConfig](#kubelet-config-k8s-io-v1-CredentialProviderConfig)
+
+
+<!-- <p>CredentialProvider represents an exec plugin to be invoked by the kubelet. The plugin is only
+invoked when an image being pulled matches the images handled by the plugin (see matchImages).</p> -->
+<p>CredentialProvider 代表 kubelet 调用的执行插件。该插件仅在
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kubelet-config.v1.md:77]: exec 插件，不要译为执行插件。
这句话翻译不完整。

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+<thead><tr><th width="30%"><!--Field-->字段</th><th><!--Description-->描述</th></tr></thead>
+<tbody>
+
+
+<tr><td><code>name</code> <B><!--[Required]-->[必需]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <!--span class="text-muted">No description provided.</span-->
+   <p>NUMA 节点</p>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kubelet-config.v1.md:239]:
```suggestion
   <p>环境变量名称</p>
```

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1.md [hunk 4]
```diff
@@ -0,0 +1,250 @@
...
+</tr>
+<tr><td><code>value</code> <B>[Required]</B><br/>
+<code>string</code>
+</td>
+<td>
+   <!--span class="text-muted">No description provided.</span-->
+   <p>NUMA 节点</p>
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/kubelet-config.v1.md:246]:
```suggestion
   <p>环境变量取值</p>
```

Comment (helen-frank) [reply content/zh-cn/docs/reference/config-api/kubelet-config.v1.md:246]: got it

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1.md [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+package: kubelet.config.k8s.io/v1
+auto_generated: true
+-->
+
+---
+title: Kubelet 配置 (v1)
+content_type: tool-reference
+package: kubelet.config.k8s.io/v1
+auto_generated: true
+---
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/kubelet-config.v1.md:13]: Line 8-13 shall be moved to the top, that is, starting from line 1.

**Review by windsonsea — COMMENTED** [review]
The preview has some problem: https://deploy-preview-42362--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/config-api/kubelet-config.v1/

File: content/zh-cn/docs/reference/config-api/kubelet-config.v1.md (hunk 5 above)
Comment (helen-frank) [reply content/zh-cn/docs/reference/config-api/kubelet-config.v1.md:13]: got it

**Comment by windsonsea** [comment]
/lgtm

Merge now and will fix some minor issues soon.

**Pull request merged**